packages/vuetify/src/components/VCard/VCard.sass

```diff
@@ -17,6 +17,8 @@
   outline: none
   text-decoration: none
   transition-property: box-shadow, opacity
+  word-break: break-word
```

Comment:
word-break is deprecated

Reply:
IE don’t think so,
https://github.com/vuetifyjs/vuetify/blob/5c61eee424f982cf2d8c9c8b9f411552101a6749/packages/vuetify/src/styles/settings/_utilities.scss#L322

Reply:
Maybe that needs updating too then haha, MDN says it's not supported at all in IE/edge and deprecated in other browsers.

Reply:
It works in IE 11 with that rule, and not working without...